app/src/main/res/layout/setup_fragment.xml

```diff
@@ -120,11 +120,25 @@
         android:layout_marginLeft="8dp"
         android:layout_marginEnd="32dp"
         android:layout_marginRight="32dp"
+        android:text="@string/setup_text_choose_your_preferred_layout"
+        android:textAppearance="@style/TextAppearance.AppCompat.Medium"
+        app:layout_constraintEnd_toStartOf="@id/ivTick2"
+        app:layout_constraintStart_toEndOf="@id/opt2Number"
+        app:layout_constraintTop_toTopOf="@id/opt2Number" />
+
+    <TextView
+        android:id="@+id/opt3Text"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="8dp"
+        android:layout_marginLeft="8dp"
+        android:layout_marginEnd="32dp"
+        android:layout_marginRight="32dp"
         android:text="@string/setup_text_choose_your_preferred_clock"
         android:textAppearance="@style/TextAppearance.AppCompat.Medium"
-        app:layout_constraintEnd_toStartOf="@+id/ivTick2"
```

Comment:
Note that the '+' is missing after the changes. 

When you declare an id for the first time you use, '+'. When you refer an already declared id, you don't need to add '+'.